engines/sludge/newfatal.cpp

```diff
@@ -41,8 +35,8 @@ namespace Sludge {
 
 const char emergencyMemoryMessage[] = "Out of memory displaying error message!";
 
-static char *fatalMessage = NULL;
-static char *fatalInfo = joinStrings("Initialisation error! Something went wrong before we even got started!", "");
```

Comment:
You shouldn't use global `Common::String` variables ( http://wiki.scummvm.org/index.php/Coding_Conventions#Use_of_global_.2F_static_variables ).